doc_src/en/OmegaT_Preferences.xml

```diff
@@ -286,7 +286,24 @@
           displayed on the same line.</para>
         </listitem>
       </varlistentry>
-    </variablelist>
+
+	  <varlistentry id="dialogs.preferences.glossary.sort.by.source.term.length">
```

Comment:
An indent is wrong because of TAB characters.

Reply:
The indent in the current document source files are not consistent among lines and files, it seems not a problem.